test/operators.test.mjs

```diff
@@ -158,3 +159,16 @@ eq 0x0 0x1
   await proveAndVerify(asm)
   t.pass()
 })
+
+test('should mod inverse in place', async t => {
```

Comment:
Could you add another test but storing the output in a different register?

Reply:
you mean, output to 0x0 instead of 0x2? added